docs/sphinx/developers/generating-test-images.txt

```diff
@@ -12,6 +12,33 @@ Whatever is before the ``&`` is the image name; remaining key value pairs
 should be pretty self-explanatory. Just replace the values with whatever
 you need for testing.
 
+Additionally, you can put such values in a separate .ini file:
+
+::
+
+    touch my-special-test-file.fake
+    echo "pixelType=uint8" >> my-special-test-file.fake.ini
+    echo "sizeX=8192" >> my-special-test-file.fake.ini
+    echo "sizeY=8192" >> my-special-test-file.fake.ini
+
+In fact, just the .fake.ini file alone suffices:
+
+::
+
+    echo "pixelType=uint8" >> my-special-test-file.fake
+    echo "sizeX=8192" >> my-special-test-file.fake
+    echo "sizeY=8192" >> my-special-test-file.fake
+
+If you include a "[GlobalMetadata]" section to the ini file,
+then all the included values will be accessible from the
+global metadata map:
+
+::
+
+    echo "[GlobalMetadata]" >> my-special-test-file.fake
+    echo "my.key=some.value" >> my-special-test-file.fake
```

Comment:
One last nitpick: ```my-special-test-file.fake``` should be ```my-special-test-file.fake.ini```, otherwise the key/value pairs won't appear in the original metadata table.